### PR TITLE
Update logs_pipelines.md

### PR DESCRIPTION
**Pull request by Xorima**
### What does this PR do?
Adding documentation for granting permissions to pipelines

### Motivation
Had to reach out to support for this information, took too long to get an answer

### CODE REVIEW

File: content/en/api/logs_pipelines/logs_pipelines.md [hunk 1]
```diff
@@ -19,3 +19,4 @@ Pipelines and processors operate on incoming logs, parsing and transforming them
 [1]: /help
 [2]: https://app.datadoghq.com/logs/pipelines
 [3]: https://docs.datadoghq.com/logs/processing
+[4]: https://docs.datadoghq.com/account_management/faq/managing-global-role-permissions/?tab=datadogussite#granting-permissions-within-limited-scopes
```
Comment (l0k0ms) [inline content/en/api/logs_pipelines/logs_pipelines.md]:
```suggestion
[4]: /account_management/faq/managing-global-role-permissions/?tab=datadogussite#granting-permissions-within-limited-scopes
```

File: content/en/api/logs_pipelines/logs_pipelines.md [hunk 2]
```diff
@@ -19,3 +19,4 @@ Pipelines and processors operate on incoming logs, parsing and transforming them
 [1]: /help
 [2]: https://app.datadoghq.com/logs/pipelines
 [3]: https://docs.datadoghq.com/logs/processing
```
Comment (l0k0ms) [inline content/en/api/logs_pipelines/logs_pipelines.md]:
```suggestion
[3]: /logs/processing
```

File: content/en/api/logs_pipelines/logs_pipelines.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ external_redirect: /api/#logs-pipelines
 
 ## Logs Pipelines
 
-<mark>The Logs Pipelines endpoints are not supported in Datadog's client libraries. To request this functionality, contact [Datadog Support][1].</mark>
+<mark>The Logs Pipelines endpoints are not supported in Datadog's client libraries. To request this functionality, contact [Datadog Support][1]. To permission your account see [Global Role Permissions][4]</mark>
```
Comment (l0k0ms) [inline content/en/api/logs_pipelines/logs_pipelines.md]:
```suggestion
<mark>The Logs Pipelines endpoints are not supported in Datadog's client libraries. To request this functionality, contact [Datadog Support][1]. To set Pipeline permission see the [Granting permissions within limited scopes][4] documentation</mark>
```

**Comment by Xorima** [comment]
resolved @l0k0ms 

note should `[2]: https://app.datadoghq.com/logs/pipelines` also be changed to be relative, and do I need to do this in any other languages or will you handle that internally?

**Comment by l0k0ms** [comment]
@Xorima nop, only docs.datadoghq.com/* links need to be relative, the app link shouldn't.

We are going to perform an audit to the rest of the doc to remove such absolute links

**Comment by l0k0ms** [comment]
Many thanks for this PR :)

**Comment by l0k0ms** [comment]
pushed the PR to fix all other link ^ #6640

**Pull request merged**